code/datums/action.dm

```diff
@@ -198,6 +198,13 @@
 /datum/action/item_action/toggle_light
 	name = "Toggle Light"
 
+/datum/action/item_action/toggle_light/Trigger()
```

Comment:
remove this part

Reply:
Why

Reply:
He is essentially adding snowflake code to toggle the light instead of making the PDA light toggle like our other light toggles work, when he could just… make the PDA light toggle function programmatically the same as other toggleable lights.

Reply:
> I agree, why?

also now that he did the other changes (so he’s 95% there) i don’t think these lines are needed